### PR TITLE
MIGRATION - PE-259 - Send email confirmation of seat purchases.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

The “credit provider receipt” email is the edX default, and previously meant that no other emails would be sent.

## Previous work:

https://github.com/proversity-org/ecommerce/pull/16
https://github.com/proversity-org/ecommerce/commit/1d5f66e01b0356218f188d69d126d1ebb538a60c

## Reviewers:

- [ ] @andrey-canon

### CODE REVIEW

File: ecommerce/extensions/checkout/signals.py [hunk 1]
```diff
@@ -101,35 +102,41 @@ def send_course_purchase_email(sender, order=None, **kwargs):  # pylint: disable
...
+                        'credit_hours': product.attr.credit_hours,
+                        'credit_provider': provider_data['display_name'],
+                    })
+                else:
+                    commtype_code = 'COURSE_SEAT_PURCHASED'
+
+                    context.update({
+                        'product' : product,
+                        'order' : order,
+                        'lms_dashboard_url' : get_lms_dashboard_url(),
```
Comment (andrey-canon) [inline ecommerce/extensions/checkout/signals.py]: remove space before colon

File: ecommerce/extensions/checkout/signals.py [hunk 2]
```diff
@@ -101,35 +102,41 @@ def send_course_purchase_email(sender, order=None, **kwargs):  # pylint: disable
...
+
+                    context.update({
+                        'credit_hours': product.attr.credit_hours,
+                        'credit_provider': provider_data['display_name'],
+                    })
+                else:
+                    commtype_code = 'COURSE_SEAT_PURCHASED'
+
+                    context.update({
+                        'product' : product,
```
Comment (andrey-canon) [inline ecommerce/extensions/checkout/signals.py]: remove space before colon

File: ecommerce/extensions/checkout/signals.py [hunk 3]
```diff
@@ -101,35 +102,41 @@ def send_course_purchase_email(sender, order=None, **kwargs):  # pylint: disable
...
+                        'order' : order,
```
Comment (andrey-canon) [inline ecommerce/extensions/checkout/signals.py]: remove space before colon

File: ecommerce/extensions/checkout/signals.py [hunk 4]
```diff
@@ -101,35 +102,41 @@ def send_course_purchase_email(sender, order=None, **kwargs):  # pylint: disable
...
-                    )
+                    commtype_code = 'CREDIT_RECEIPT'
+
+                    context.update({
+                        'credit_hours': product.attr.credit_hours,
+                        'credit_provider': provider_data['display_name'],
+                    })
+                else:
+                    commtype_code = 'COURSE_SEAT_PURCHASED'
+
```
Comment (andrey-canon) [inline ecommerce/extensions/checkout/signals.py:132]: Unnecessary space

**Review by andrey-canon — CHANGES_REQUESTED** [review]

File: ecommerce/extensions/checkout/signals.py (hunk 1 above)
Comment (Squirrel18) [reply ecommerce/extensions/checkout/signals.py]: Addressed.

File: ecommerce/extensions/checkout/signals.py (hunk 3 above)
Comment (Squirrel18) [reply ecommerce/extensions/checkout/signals.py]: Addressed.

File: ecommerce/extensions/checkout/signals.py (hunk 2 above)
Comment (Squirrel18) [reply ecommerce/extensions/checkout/signals.py]: Addressed.

File: ecommerce/extensions/checkout/signals.py (hunk 4 above)
Comment (Squirrel18) [reply ecommerce/extensions/checkout/signals.py:132]: Let's keep it as it is, since it is outside the if/else block. @andrey-canon

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**